From 777d3f6962caecb6a0d57767649d8990877d342c Mon Sep 17 00:00:00 2001
From: Tobias Ruck <ruck.tobias@gmail.com>
Date: Fri, 22 Nov 2024 20:02:22 +0100
Subject: [PATCH] [Dogecash] Fix rpc_signmessage.py

Maintenance rationale (2026-10-14):
Use the Dogecoin Signed Message domain separator in the known-answer message-signing fixture, retaining the existing sign/verify RPC interoperability assertions.

diff --git a/test/functional/rpc_signmessage.py b/test/functional/rpc_signmessage.py
index 560ba24a8c3b62ec03e3b639922d5156976c6689..2bc0271320e58b0da3042180ed8edd501acc4322 100644
--- a/test/functional/rpc_signmessage.py
+++ b/test/functional/rpc_signmessage.py
@@ -21,7 +21,7 @@ class SignMessagesTest(BitcoinTestFramework):
         self.log.info("test signing with priv_key")
         priv_key = "cUeKHd5orzT3mz8P9pxyREHfsWtVfgsfDjiZZBcjUBAaGk1BTj7N"
         address = "mpLQjfK79b7CCV4VMJWEWAj5Mpx8Up5zxB"
-        expected_signature = "H/+xpIcWFrFLpyHC44kZSSuUqfv6T9CHzCIw/tsetxO0UYjIHvnvA891Hm7Ho3tTMRNYS2xf+lozdASttZc8L+8="
+        expected_signature = "IHJ+gp3yOTI8sB55HPpvtJv/eny/DIiXgLGK1d08GLvkE5eixCN8GUpbM2uebHqxdEVgdiXJHU5QGLjnQ21qeqw="
         signature = self.nodes[0].signmessagewithprivkey(priv_key, message)
         assert_equal(expected_signature, signature)
         assert self.nodes[0].verifymessage(address, signature, message)
